runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -96,7 +96,7 @@ public String toString()
 		.build();
 
 	// Notifier properties
-	private static final Color FLASH_COLOR = new Color(255, 0, 0, 70);
+	public static  Color FLASH_COLOR = new Color(255, 0, 0, 70);
```

Comment:
This should not be static if it is changed to be an instance field.
```suggestion
	public Color FLASH_COLOR = new Color(255, 0, 0, 70);
```